USER_MANUAL.md

```diff
@@ -385,5 +385,14 @@ Manual steps:<br>
   1. Start the gym again and note that the level actors should be at their previous shutdown positions, not their original positions. You can do this repeatedly.
   1. The test has now passed. Don’t forget to revert the two settings changes you made before you run another test.
 
+##### Async Package Loading Gym
+Tests that async package loading works when activated.<br>
```

Comment:
This could do with an explanation of how it tests that- is it a client with async loading enabled connects and doesn't create an actor until the asset is loaded? Or actually I thought it was both a server and client thing.

Reply:
Yes, it tests that a client both doesn't have the class loaded in memory after begin play has been called, and manages to load an instance of that class at some later point in it's lifetime. This is definitely a thing on servers and clients, and this same logic could be run by a second non-auth server. I just picked a client because they're easier to spin up and attach to a deployment.